docs/faq.md

```diff
@@ -271,6 +273,80 @@ There are several ways of doing this:
 - [`docker pull`](https://docs.docker.com/engine/reference/commandline/pull/) the image built with Jib to have it available in your local Docker daemon.
 - Alternatively, instead of using a Docker daemon, you can run a local container registry, such as [Docker registry](https://docs.docker.com/registry/deploying/) or other repository managers, and point Jib to push to the local registry.
 
+### What would a Dockerfile for a Jib-built image look like?
+
+A Dockerfile that performs a Jib-like build is shown below:
+
+```Dockerfile
+# Jib uses distroless java as the default base image
+FROM gcr.io/distroless/java:latest
+
+# Multiple copy statements are used to break the app into layers, allowing for faster rebuilds after 
+# small changes
+COPY dependencyJars /app/libs
+COPY resources /app/resources
+COPY classFiles /app/classes
+COPY src/main/extra-dir /
+
+# Sets the app as the container entrypoint with 2 JVM flags and 2 main arguments. Also adds the
+# appropriate files to the classpath.
+ENTRYPOINT ["java", "-Xms512m", "-Xdebug", "-cp", "/app/resources:/app/classes:/app/libs/*", "SomeMainClass"]
```

Comment:
This implies that we always configure `-Xms512 -Xdebug` How about `["java", container.jvmFlags, "-cp", "/app/...etc", container.mainClass]`